packages/react-native/ReactCommon/react/renderer/imagemanager/platform/ios/react/renderer/imagemanager/RCTImagePrimitivesConversions.h

```diff
@@ -102,13 +122,30 @@ inline static NSURLRequest *NSURLRequestFromImageSource(const facebook::react::I
 
   NSMutableURLRequest *request = [[NSMutableURLRequest alloc] initWithURL:url];
 
-  /*
-  // TODO(shergin): To be implemented.
-  request.HTTPBody = ...;
-  request.HTTPMethod = ...;
-  request.cachePolicy = ...;
-  request.allHTTPHeaderFields = ...;
-  */
+  NSString *method = @"GET";
+  if (!imageSource.method.empty()) {
+    method = [[NSString alloc] initWithBytesNoCopy:(void *)imageSource.method.c_str()
+                                            length:imageSource.method.size()
+                                          encoding:NSUTF8StringEncoding
+                                      freeWhenDone:NO]
+                 .uppercaseString
```

Comment:
Emm, I used `no copy` method because I think it can reduce one copy, if we use `initWithUTF8String`, it would produces twice copy operations from c_str to nsadata.